curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/send-and-display-chat-messages.md

```diff
@@ -22,9 +22,9 @@ Within the form submit code, you should emit an event after you define `messageT
 socket.emit('chat message', messageToSend);
 ```
 
-Now, on your server, you should be listening to the socket for the event `'chat message'` with the data being named `message`. Once the event is received, it should emit the event `'chat message'` to all sockets `io.emit` with the data being an object containing `name` and `message`.
+Now, on your server, you should be listening to the socket for the event `'chat message'` with the data being named `message`. Once the event is received, it should emit the event `'chat message'` to all sockets `io.emit` with the data being an object containing `username` and `message`.
```

Comment:
Id change the last sentence to:
Once the event is received, it should emit the event `'chat message'` to all sockets using `io.emit`, sending a data object containing the `username` and `message`.

Reply:
I think your sentence is shorter and easier to read. Should I add this change too?